### PR TITLE
s/f/other/

### PR DESCRIPTION
**Pull request by kzidane**
Addresses @brianyu28's [comment](https://github.com/cs50/submit50/commit/ef5bfba3751a0d055e967c7c75bf5be883a81096#commitcomment-24869782).

### CODE REVIEW

**Comment by kzidane** [comment]
@dmalan `--exclude-standard` looks for `.git/info/exclude` not `$GIT_DIR/info/exclude` apparently.

File: submit50.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def submit(org, branch):
 
     # get file lists
     files = run("git ls-files").splitlines()
-    others = run("git ls-files --exclude-standard --other").splitlines()
+    others = run("git ls-files --exclude-from={}/info/exclude --other".format(run.GIT_DIR)).splitlines()
```
Comment (dmalan) [inline submit50.py]: Why this change? Already excluding per https://github.com/cs50/submit50/blob/c4c41de8558f99338dbbb919616753cce9cc2341/submit50.py#L547?
